ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -14,6 +14,38 @@
     </span>
 
     <span class="navbar-right">
+      <!-- Notifications Menu -->
+      <span class="dropdown" ng-controller="NotificationCtrl">
+        <button id="notificationLabel" title="Recent Notifications" role="button"
```

Comment:
Fwiw I think we've generally done sentence case for tooltip titles. eg.

`Recent notifications`

Reply:
agreed, yeah